bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/CcuGateway.java

```diff
@@ -151,7 +149,10 @@ protected void addChannelDatapoints(HmChannel channel, HmParamsetType paramsetTy
 
     @Override
     protected void setVariable(HmDatapoint dp, Object value) throws IOException {
-        String strValue = StringUtils.replace(ObjectUtils.toString(value), "\"", "\\\"");
+        String strValue = "";
+        if (value != null) {
+            strValue = value.toString().replace("\"", "\\\"");
```

Comment:
In case you have concerns about performance of String.replace() see https://stackoverflow.com/questions/16228992/commons-lang-stringutils-replace-performance-vs-string-replace

Reply:
This was a concern in java 8, but the performance gap was largely fixed by java 11, which is what we are now using.